### ISSUE TITLE
peer review --- cx223

### ISSUE THREAD

**Issue by ChengnanXu** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#3]
This group plans to explore and evaluate the existing performance of algorithms and proposed a new method to identify an optimal learning rate, .to help boost the efficiency and accuracy of training neural network. Although there are many hyper-parameters need to be considered when fitting the algorithms brought many challenges in the field of neural network processing, the learning rate is the most, thus it is imperative to find new methods to quickly help figuring out the optimal learning rate to avoid divergence problems. They will accomplish this goal by studying existing algorithms for adaptive learning rate, exploring the performance of these algorithms and developing new ways to optimize the learning rate to help improve part of the algorithms performances.
### Three cool things like the most are:
1.	They provide a detailed overview of the needs to improving the neural network performance to help solving the real-world problems. They provided detailed reasons of this project significant and practical.
2.	Their study will provide better method to help determining the learning rates and selecting models, making them more accuracy and efficiency. They also will based on various existing methods, algorithms and resources, explained very well in timeline what they will do, which will help them done project in high efficiency.
3.	They also detailed introduce the feasible by stating that there have various existing work, group members have experiences, etc. Thus, their results would be very promising.
### Three areas for improvement:
4.	They could explain in more how they will to evaluate their results of learning rate and how they compare with different existing methods of algorithms in different fields, is their have some standard.
5.	It would be helpful to explain more in-depth about what kinds of dataset they want to use and give some detailed explanation. Since the neural network always need much more dataset and hard computational methods there isn't much information on how this possible risk could be managed.
6.	They could illustrate some application areas of their project, based on neural network,  to help readers who unfamiliar with the topic understand the project. It would be helpful if they list fields that will benefit the most from their optimized learning rate within neural network.
